msprime/likelihood.py

```diff
@@ -28,9 +28,15 @@ def log_mutation_likelihood(ts, mutation_rate):
     """
     Returns the unnormalised log probability of the stored pattern of mutations
     on the stored tree sequence, assuming infinite sites mutation. In particular,
-    each stored site must only contain a single mutation. The omitted normalising
-    constant depends on the pattern of mutations, but not on the tree sequence or
-    the mutation rate.
+    each stored site must only contain a single mutation, and all mutant alleles
+    are treated interchangeably as having arisen from a single mutation rate.
```

Comment:
Do we want to say e.g. "(this is automatically true for some msprime mutation models, such as JC69)"

Reply:
Good idea, I'll add that in.